GOVERNANCE.md

```diff
@@ -82,6 +82,87 @@ Once the Call for Agreement voting period has elapsed, all votes are counted, wi
 
 Once rejected, a Call for Agreement must be revised before re-submission for a subsequent vote. All rejected Calls for Agreement will be reported to the Operator Framework Community as rejected.
 
+### Project Governance
+
+All Operator Framework projects must use Apache 2.0 license and align with the mission, scope, and code of conduct
+of the Operator Framework defined by this Operator Framework charter.
+
+Projects in the Operator Framework fall into two categories: Incubating and Graduated
+
+#### Incubating Projects
+
+In order to foster community growth and innovation in the development and administration of Kubernetes operators, the
+Operator Framework admits Incubating projects into the Operator Framework. These projects have a lower barrier to entry
+and do not include any support, maintenance, or production guarantees. They enable new ideas to gain traction
+and maturity.
+
+Criteria:
+
+1. Must have at least one active project maintainer who is a member of the Operator Framework community documented in
```

Comment:
There should be a defined `OWNERS_ALIAS` alias for this [community role](https://github.com/operator-framework/community/pull/12/files#diff-a28bb6e2763f5f03cc80a7052e486a71R173), ex. `admins`.